listenbrainz/webserver/views/stats_api.py

```diff
@@ -151,3 +147,16 @@ def _is_valid_range(stats_range):
         result (bool): True if given range is valid
     """
     return stats_range in StatisticsRange.__members__
+
+
+def _get_stats_for_range(data: UserArtistStatRow, stats_range: StatisticsRange) -> UserArtistStatForRange:
```

Comment:
docs/tests?

Reply:
Added docs to a similar function after rebase, there are pretty extensive tests testing this codepath already.